.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  extends: ['eslint:recommended', 'prettier'],
+  ignorePatterns: ['dist'],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    allowImportExportEverywhere: true,
+  },
+  rules: {
+    'no-console': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'warn' : 'off',
+    'no-undef': 'off',
```

Comment:
Do we need to allow `undefined` variables?

Reply:
Turning it on throws a whole lot of warnings about `document` and `window` and things of the nature since they aren't explicitly defined. This config is also copied from our ui repo and it seems to be useful for now